### PR TITLE
cb 4004 Include Kibana Query URL into the API E2E Test Result Reports

### PR DESCRIPTION
**Pull request by afarsang**
Include Kibana Query URL into the API E2E Test Result Reports
Closes CB-4004

### CODE REVIEW

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/database/RedbeamsDatabaseTestDto.java [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
+import com.sequenceiq.it.cloudbreak.dto.DeletableRedbeamsTestDto;
+import com.sequenceiq.it.cloudbreak.search.Searchable;
 import com.sequenceiq.it.cloudbreak.util.ResponseUtil;
 import com.sequenceiq.redbeams.api.endpoint.v4.database.DatabaseV4Endpoint;
 import com.sequenceiq.redbeams.api.endpoint.v4.database.request.DatabaseV4Request;
 import com.sequenceiq.redbeams.api.endpoint.v4.database.responses.DatabaseV4Response;
 
+// import java.util.function.BiConsumer;
+// import java.util.function.Function;
+// import javax.ws.rs.WebApplicationException;
```
Comment (doktoric) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/dto/database/RedbeamsDatabaseTestDto.java]: can we remove these lines ?

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/listener/ReportListener.java [hunk 2]
```diff
@@ -1,14 +1,108 @@
...
+        TestContext testContext;
+        Object[] parameters = tr.getParameters();
+        if (parameters == null || parameters.length == 0) {
+            LOGGER.info("!!!ez");
+            return;
+        }
+        try {
+            testContext = (TestContext) parameters[0];
+        } catch (ClassCastException e) {
+            LOGGER.info("!!!ez inkabb");
```
Comment (doktoric) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/listener/ReportListener.java]: :D please remove this line

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/listener/ReportListener.java [hunk 3]
```diff
@@ -1,14 +1,108 @@
...
+
+            return;
+        }
+        Iterable<Searchable> searchables = Iterables.filter(testContext.getResources().values(), Searchable.class);
+        List<Searchable> listOfSearchables = StreamSupport.stream(searchables.spliterator(), false).collect(Collectors.toList());
+        if (listOfSearchables.size() == 0) {
+            LOGGER.info("!!!vagy ez");
```
Comment (doktoric) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/listener/ReportListener.java]: :D :D

Comment (doktoric) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/listener/ReportListener.java]: made my day

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/listener/ReportListener.java [hunk 4]
```diff
@@ -1,14 +1,108 @@
...
+            return;
+        }
+        SearchUrl searchUrl = new KibanaSearchUrl();
+        tr.getTestContext().setAttribute(tr.getName() + SEARCH_URL,
+                searchUrl.getSearchUrl(listOfSearchables, new Date(tr.getStartMillis()), new Date(tr.getEndMillis())));
+    }
+
+//    public void logKibanaUrl(ITestResult tr) {
```
Comment (doktoric) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/listener/ReportListener.java]: can we remove these lines if not required

Comment (afarsang) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/listener/ReportListener.java]: yes

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/listener/ReportListener.java (hunk 3 above)
Comment (afarsang) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/listener/ReportListener.java]: oops :)

**Pull request merged**